packages/x-charts/src/locales/frFR.ts

```diff
@@ -0,0 +1,10 @@
+import { ChartsLocaleText } from './utils/chartsLocaleTextApi';
+import { getChartsLocalization } from './utils/getChartsLocalization';
+
+const frFRCharts: Partial<ChartsLocaleText> = {
+  // No group
```

Comment:
What do you mean by "no group"?

Reply:
I added one to remove this default comment